ocean_provider/utils/basics.py

```diff
@@ -161,6 +164,24 @@ def get_web3_connection_provider(
         raise AssertionError(msg)
 
 
+def get_network_name(chain_id: int) -> str:
+    if not chain_id:
+        logger.error("Chain ID is missing")
+
+    if chain_id == 8996:
+        return "Ganache"
+
+    address_path = Path(os.path.join(addresses.__file__, "..", "address.json"))
+
+    address_file = address_path.expanduser().resolve()
+    with open(address_file) as f:
+        addresses_json = json.load(f)
+
+    for k, v in addresses_json.items():
+        if v["chainId"] == chain_id:
+            return k
+
```

Comment:
```suggestion
return ("Unknown")
```

Reply:
maybe provider works on a custom network. let's return "Unknown" if that's the case